src/dependency-container.ts

```diff
@@ -96,8 +97,8 @@ export class DependencyContainer implements Types.DependencyContainer {
     const registration = this.getRegistration(token);
 
     if (!registration) {
-      if (typeof (token) === "string") {
-        throw `Attempted to resolve unregistered dependency token: ${token}`;
+      if (isNormalToken(token)) {
+        throw `Attempted to resolve unregistered dependency token: ${token.toString()}`;
```

Comment:
`.toString()` not needed within a template literal.

Reply:
If `token` is `symbol` , it will throw an error.

```javascript
Symbol() + ""
//VM148:1 Uncaught TypeError: Cannot convert a Symbol value to a string
//    at <anonymous>:1:10
```

Reply:
Fair point, did not know that! =]